FHICORC.Backend/FHICORC.Tests.UnitTests/DGCGComponentTests/DGCGRevocationServiceComponentTests.cs

```diff
@@ -63,40 +67,110 @@ public void Setup()
             }
         }
 
+        private List<string> GetRandomHashesInLists(int newHashAmount)
+        {
+            var hashList = new List<string>();
+
+            for (int i = 0; i < newHashAmount; i++)
+            {
+                hashList.Add(i.ToString());
+            }
+            return hashList;
+        }
+
+        [TestCase(999, 1), Order(1)]
+        public void UploadHashesTestAddOneBatch(int newHashAmount, int addedBatches)
+        {
+            // Arrange
```

Comment:
There should only be one arrange, act, and assert stage. Move them into the same stage, or create 2 unit tests. Do this for all the unit tests please.